lib/rules/prefer-stateless-function.js

```diff
@@ -244,6 +276,12 @@ module.exports = {
     }
 
     return {
+      ClassDeclaration: function (node) {
+        if (utils.isPureComponent(node) && ignorePureComponents === true) {
```

Comment:
I don't think we need the `=== true` :-) also we should probably check the boolean first to avoid the possibly unnecessary isPureComponent call.